app/src/main/java/app/revanced/manager/ui/component/GenericDeleteDialog.kt

```diff
@@ -0,0 +1,41 @@
+package app.revanced.manager.ui.component
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Delete
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import app.revanced.manager.R
+
+@Composable
+fun GenericDeleteDialog(
+    onDismiss: () -> Unit,
+    onConfirm: () -> Unit,
+    title: @Composable () -> Unit,
+    description: @Composable () -> Unit
```

Comment:
Maybe these should take a string resource instead. Dialogs that need custom composable code can use `AlertDialog` directly.